bookkeeper-server/src/main/java/org/apache/bookkeeper/client/BookKeeper.java

```diff
@@ -859,9 +859,7 @@ public void asyncCreateLedger(final int ensSize,
     public void asyncCreateLedger(final int ensSize, final int writeQuorumSize, final int ackQuorumSize,
                                   final DigestType digestType, final byte[] passwd,
                                   final CreateCallback cb, final Object ctx, final Map<String, byte[]> customMetadata) {
-        if (writeQuorumSize < ackQuorumSize) {
-            throw new IllegalArgumentException("Write quorum must be larger than ack quorum");
-        }
+        checkLedgerCreationParameters(ensSize, writeQuorumSize, ackQuorumSize);
```

Comment:
It will throw exceptions directly, the user may need to try catch `asyncCreateLedger`.
How about `cb.createComplete` with an error code?

Reply:
same question

Reply:
+1